eval_llm/MMLU/run_mmlu_opennmt.py

```diff
@@ -7,13 +7,14 @@
 import time
 import pandas as pd
 from onmt.utils.logging import init_logger
-from onmt.utils.distributed import MPInference
+from onmt.inference_engine import InferenceEngine
```

Comment:
I think we miss the new `inference_engine` file in this commit :upside_down_face: